doc/man3/OSSL_PARAM_int.pod

```diff
@@ -264,6 +269,17 @@ OSSL_PARAM_set_octet_ptr() sets the OCTET string pointer in the parameter
 referenced by B<p> to the values B<val>.
 The length of the OCTET string is provided by B<used_len>.
 
+OSSL_PARAM_get_utf8_string_ptr() retrieves the pointer to a UTF8 string from
+the parameter pointed to by B<p>, and stores that pointer in B<*val>.
+This is different from OSSL_PARAM_get_utf8_string_ptr(), which copies the
+string.
+
+OSSL_PARAM_get_ooctet_string_ptr() retrieves the pointer to a octet string
```

Comment:
ooooooctect ?

Reply:
My keyboard is showing age...  some keyclicks give me double value, and I don't always notice...